tpl/editView.html

```diff
@@ -1,7 +1,7 @@
 <form>
 	<ul class="flat">
 		<li class="sep">Label *</li>
-    <li class="input"><input type="text" name="label" placeholder="A label to display" value="{{= it.label || '' }}" /></li>
+    <li class="input"><input type="text" name="label" placeholder="A label to display" value="{{! it.label || '' }}" /></li>
```

Comment:
Try the problem recipe I described in my [PR comment](https://github.com/devgeeks/Encryptr/pull/175#issuecomment-127475500) - these changes resolved the problems in the label and field values, respectively.